tailwind/styles.css

```
@@ -1,1015 +1,110 @@
-    grid-template-columns: 34px minmax(0, 1fr) auto;
-    min-height: 86px;
-    padding: 12px;
-  }
-}
+@source "../templates/**/*.gohtml";
+@source "../static/atomic-handheld.js";
+
+/* Approved Atomic mockup: preserve the handheld scene rules. */
+
+
+#dw-atomic{color-scheme:light dark;--surface:light-dark(#eef2ee,#22332d);--ink:light-dark(#253c35,#e8eee7);--soft:light-dark(#5c7168,#b5c7bb);--accent:light-dark(#306557,#b1d0a8);--edge:light-dark(#bdcec1,#627c68);font:400 15px/1.7 'Chivo',sans-serif;color:var(--ink);width:100%}
```

Stylelint (17.14.0)

[error] Expected no quotes around "Chivo" (font-family-name-quotes)

(font-family-name-quotes)